src/pages/ticket.astro

```diff
@@ -0,0 +1,214 @@
+            document.body.appendChild(downloadLink);
+
+            // Trigger a click on the link to start the download
+            downloadLink.click();
+
+            // Remove the link from the document
+            document.body.removeChild(downloadLink);
+
+            // Restore the original background image after capturing
+            elementToCapture.style.backgroundImage = "";
+        });
+    });
+    // document.getElementById("down-ticket").addEventListener("click", function() {
+    //     const elementToCapture = document.getElementById("ticket-card");
+
+    //     // Use html2canvas to capture the content as an image with higher resolution and a transparent background
+    //     html2canvas(elementToCapture, { quality: 1.0, scale: 2, backgroundColor: "transparent" }).then(function(canvas) {
+    //         // Convert the canvas content to a data URL (JPEG format with default compression)
+    //         const dataURL = canvas.toDataURL("image/jpeg", 0.8); // 0.8 is the quality parameter (0.0 to 1.0)
+
+    //         // Open Instagram with the image ready for sharing
+    //         window.location.href = `instagram://library?AssetPath=${encodeURIComponent(dataURL)}`;
+    //     });
+    // });
+</script>
+
+<Layout title="Welcome to Astro.">
+	<main>
+		<div class="container">
+      <div class="card" id='ticket-card'>
+          <div class="username-box">
+            <p class="text-template">Username</p>
+          </div>
+          <img src="public/ดาวน์โหลด-03 5.svg" alt="" />
+          <div class="mbti-box">
+            <p class="text-template">MBTI</p>
+          </div>
+          <div class="character-box">
+            <img src="public/IMG_9059 2.png" alt="">
+          </div>
+      </div>
+      		<a class="download-box" id="down-ticket">
+       			<i class="gg-software-download"></i>
+        		<p class="text-down">Download</p>
+      		</a>
+    	</div>
+	</main>
+</Layout>
+
+<style>
+@import url('https://fonts.cdnfonts.com/css/zilla-slab');	
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+.gg-software-download {
+    box-sizing: border-box;
+    position: relative;
+    display: block;
+    transform: scale(var(--ggs,1));
+    width: 16px;
+    height: 6px;
+    border: 2px solid;
+    border-top: 0;
+    border-bottom-left-radius: 2px;
+    border-bottom-right-radius: 2px;
+    margin-top: 8px
+}
+
+.gg-software-download::after {
+    content: "";
+    display: block;
+    box-sizing: border-box;
+    position: absolute;
+    width: 8px;
+    height: 8px;
+    border-left: 2px solid;
+    border-bottom: 2px solid;
+    transform: rotate(-45deg);
+    left: 2px;
+    bottom: 4px
+}
+
+.gg-software-download::before {
+    content: "";
+    display: block;
+    box-sizing: border-box;
+    position: absolute;
+    border-radius: 3px;
+    width: 2px;
+    height: 10px;
+    background: currentColor;
+    left: 5px;
+    bottom: 5px
+}
+body {
+  font-family: 'Zilla Slab', sans-serif;
```

Comment:
using a selector for `body` is a little too unspecific, probably a better idea to put it in a more specific place to [avoid style conflicts](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity).